develop/rendering/hud.md

```diff
@@ -7,23 +7,23 @@ authors:
 
 # Rendering in the Hud {#rendering-in-the-hud}
 
-We already briefly touched on rendering things to the hud in the [Basic Rendering Concepts](./basic-concepts) page and [Using The Drawing Context](./draw-context), so on this page we'll stick to the `HudRenderCallback` event and the `deltaTick` parameter.
+We already briefly touched on rendering things to the hud in the [Basic Rendering Concepts](./basic-concepts) page and [Using The Drawing Context](./draw-context), so on this page we'll stick to the `HudRenderCallback` event and the `tickCounter` parameter.
 
 ## HudRenderCallback {#hudrendercallback}
 
 The `HudRenderCallback` event - provided by Fabric API - is called every frame, and is used to render things to the HUD.
 
-To register to this event, you can simply call `HudRenderCallback.EVENT.register` and pass in a lambda that takes a `DrawContext` and a `float` (deltaTick) as parameters.
+To register to this event, you can simply call `HudRenderCallback.EVENT.register` and pass in a lambda that takes a `DrawContext` and a `RenderTickCounter` (tickCounter) as parameters.
 
 The draw context can be used to access the various rendering utilities provided by the game, and access the raw matrix stack.
 
 You should check out the [Draw Context](./draw-context) page to learn more about the draw context.
 
-### DeltaTick {#deltatick}
+### Tick Delta {#deltatick}
 
-The `deltaTick` refers to the time since the last frame, in seconds. This can be used to make animations and other time-based effects.
+Frames are rendered faster than the game ticks. Tick delta is a `float` from `0.0` to `1.0` that represents the "progress" between the last game tick and the next game tick. This can be used to make animations and other time-based effects.
 
-For example, let's say you want to lerp a color over time. You can use the `deltaTickManager` to get the deltaTick, and store it over time to lerp the color:
+For example, let's say you want to lerp a color over time. You can use the `tickCounter` to get the tick delta, and store it over time to lerp the color:
```

Comment:
Same here, storing the sum of tick deltas doesn't provide framerate-independentness either. A correct usecase would be lerping a color from last tick to the color on the current tick

Reply:
Changes reverted.